dotcom-rendering/src/web/components/Standfirst.tsx

```diff
@@ -196,7 +199,7 @@ export const Standfirst = ({ format, standfirst }: Props) => {
 	return (
 		<div
 			css={[
-				nestedStyles(palette),
+				nestedStyles(format, palette),
 				standfirstStyles(format, palette),
```

Comment:
For a subsequent PR, but worth noting that if you pass `format`, you can omit `palette`, which can be inferred via `decidePalette(format)`!